### DISCUSSION TITLE
GitHub Organisation Teams

### DISCUSSION THREAD

**Discussion by VideoBSO** [bitfocus/companion#2611] [Ideas]
### Is this a feature relevant to companion itself, and not a module?

- [X] I believe this to be a feature for companion, not a module

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

Technically this is not a companion feature, but a "feature" of the github organisation:
It would be awesome if you could add a team "modules" for your organisation members, so when opening a pull request one could [@-mention](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams) the team with `@bitfocus/modules` for requesting a review.

### Usecases

Whenever opening a Pull Request for a module it in not clear to the contributor, how to get the attention of the maintainers.

**Comment by Julusian** [comment]
There is a team called `bitfocus/companion-module-developers`, which looks to be as public as it can be (https://github.com/orgs/bitfocus/teams/companion-module-developers). But I'm not sure if anyone can tag them.

But I'm not sure how a team would help here? contributors would still need to know to tag the team. And in many modules where there are active maintainers pr reviews would be best left to them than the core module maintainers.
